flight/Libraries/sanitycheck.c

```diff
@@ -211,30 +207,35 @@ DONT_BUILD_IF(MANUALCONTROLSETTINGS_STABILIZATION1SETTINGS_NUMELEM != MANUALCONT
  * @param[in] index Which stabilization mode to check
  * @returns SYSTEMALARMS_CONFIGERROR_NONE or SYSTEMALARMS_CONFIGERROR_MULTIROTOR
  */
-static int32_t check_stabilization_settings(int index, bool multirotor,
-		SharedDefsThrustModeStabBankOptions thrust_mode)
+static int32_t check_stabilization_settings(int index, bool multirotor)
 {
 	uint8_t modes[MANUALCONTROLSETTINGS_STABILIZATION1SETTINGS_NUMELEM];
+	uint8_t thrust_mode;
 
 	// Get the different axis modes for this switch position
 	switch(index) {
 		case 1:
 			ManualControlSettingsStabilization1SettingsGet(modes);
+			ManualControlSettingsStabilization1ThrustGet(&thrust_mode);
 			break;
 		case 2:
 			ManualControlSettingsStabilization2SettingsGet(modes);
+			ManualControlSettingsStabilization2ThrustGet(&thrust_mode);
 			break;
 		case 3:
 			ManualControlSettingsStabilization3SettingsGet(modes);
+			ManualControlSettingsStabilization3ThrustGet(&thrust_mode);
 			break;
 		default:
 			return SYSTEMALARMS_CONFIGERROR_NONE;
 	}
 
 	// For multirotors verify that nothing is set to "disabled" or "manual"
-	if (multirotor) {
+	if (multirotor &&
+			(thrust_mode != MANUALCONTROLSETTINGS_STABILIZATION1THRUST_FLIPOVERMODE) &&
```

Comment:
Manual mode is now allowed (encouraged, even) for flipover mode on multirotor.  So allow that case.